docker-compose.yml

```diff
@@ -4,6 +4,11 @@ services:
     container_name: palworld-dedicated-server
     image: jammsen/palworld-dedicated-server:latest
     restart: unless-stopped
+    logging:
+      driver: "local"
```

Comment:
why not json?

Reply:
It is more space efficient, and the recommended driver.